jscomp/bsb/bsb_clean.ml

```diff
@@ -64,10 +56,4 @@ let load_import_map proj_dir =
 
 let clean ~dune_args proj_dir =
   load_import_map proj_dir;
-  clean_self ~dune_args proj_dir;
-  let queue = Bsb_build_util.walk_all_deps proj_dir in
```

Comment:
Turns out we don't need to clean anything in dependency directories if we never write anything there.